packages/ifpack2/src/Ifpack2_Details_Chebyshev_decl.hpp

```diff
@@ -656,13 +663,14 @@ class Chebyshev : public Teuchos::Describable {
   /// \param D_inv [in] Vector to use as implicit right scaling of A.
   /// \param numIters [in] Maximum number of iterations of the power
   ///   method.
+  //  \param tol [in] Tolerance of the power method.
```

Comment:
Sorry, you mean from the Users guide? I had to remove few lines in the parameter description in the `_decl.hpp` file as they were wrong.